core/templates/dev/head/css/oppia.css

```diff
@@ -2182,11 +2182,15 @@ pre.oppia-pre-wrapped-text {
 */
 .oppia-rte-editor > p,
 .form-control.oppia-rte-content > div > p {
-  line-height: 28px;
+  line-height: 1.5;
   margin-bottom: 18px;
   margin-top: 18px;
 }
 
+.oppia-rte-content > div > p {
```

Comment:
Two questions:

(a) Have you tested this manually (for RTEs in content, feedback, hints, solutions, etc.)? If not, please do so and add screenshots to the comment thread of this PR.
(b) Why do we need this rule when we already have a more specific rule on line 2184? Is that rule too specific? If so, why do we not add the current selector to line 2185?